src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
+        for (int i = 0; i < 3; i++) {
```

Comment:
make it a class level constant with proper name
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
not fixed